os-server/src/main/java/app/openschool/common/security/SecurityConfigurationDisable.java

```diff
@@ -16,3 +16,4 @@ public void configure(WebSecurity web) throws Exception {
     web.ignoring().antMatchers("/**");
   }
 }
+
```

Comment:
@Avedis92 jan try to not push this kind of changes, it's making the code review harder